src/django_idom/hooks.py

```diff
@@ -30,3 +39,145 @@ def use_websocket() -> IdomWebsocket:
     if websocket is None:
         raise RuntimeError("No websocket. Are you running with a Django server?")
     return websocket
+
+
+_REFETCH_CALLBACKS: DefaultDict[FunctionType, set[Callable[[], None]]] = DefaultDict(
+    set
+)
+
+
+_Data = TypeVar("_Data")
+_Params = ParamSpec("_Params")
+
+
+def use_query(
+    query: Callable[_Params, _Data],
+    *args: _Params.args,
+    fetch_deferred_fields: bool = True,
```

Comment:
This arg causes the following Pylance warning:

Keyword parameter "fetch_deferred_fields" cannot appear in signature after ParamSpec args parameter
Pylance [reportGeneralTypeIssues](https://github.com/microsoft/pylance-release/blob/main/DIAGNOSTIC_SEVERITY_RULES.md#diagnostic-severity-rules)

Reply:
Dang. Well, the only way to work around that is to add a second function which implements the `fetch_deferred_fields=False` behavior. I can't really think of a good name for it though.

Reply:
I don't think this option should even exist. Not fetching deferred fields can completely break the ORM across all of Django, so I don't see any value in the toggle.